package.json

```diff
@@ -1364,6 +1374,11 @@
         {
           "command": "gistpad.renameRepositoryFile",
           "when": "resourceScheme == repo"
+        },
+        {
+          "command": "gistpad.syncGist",
```

Comment:
This command should be called `gistpad.syncGistFile` since it's only syncing the active file, and not all files within a file.